api/v2/openapi.yaml

```diff
@@ -410,6 +454,21 @@ definitions:
           annotations:
             $ref: '#/definitions/labelSet'
       - $ref: '#/definitions/alert'
+  alertGroups:
+    type: array
+    items:
+      $ref: '#/definitions/alertGroup'
+  alertGroup:
+    type: object
+    properties:
+      labels:
+        $ref: '#/definitions/labelSet'
+      receiver:
+        type: string
```

Comment:
What do you think of reusing the `receiver` definition below?

Why is `receiver` an object? I thought we might want to expose extra info about a single receiver in the future.

Reply:
Hm, any idea what we might expose? A receiver is mostly connection information that we don't want to expose.

Reply:
Not aware of anything specific. I am fine with either way, I would just prefer to be consistent.

Reply:
Unfortunately, I'm fine either way too :) if you want to add the extra info let me know, otherwise i'll leave it as-is

Reply:
Then I would prefer reusing the receiver definition below. (We might want to expose the receiver type in the future.)